include/realizations/config/formulation.hpp

```diff
@@ -0,0 +1,135 @@
+#ifndef NGEN_REALIZATION_CONFIG_FORMULATION_H
+#define NGEN_REALIZATION_CONFIG_FORMULATION_H
+
+#include <boost/property_tree/ptree.hpp>
+#include <string>
+
+#include "JSONProperty.hpp"
+
+namespace realization{
+  namespace config{
+
+  struct Formulation{
+    std::string type;
+    //Formulation parameters, object as a PropertyMap
+    geojson::PropertyMap parameters;
+    //List of nested formulations (used for multi bmi representations)
+    std::vector<Formulation> nested;
+
+    /**
+     * @brief Construct a new default Formulation object
+     * 
+     * Default objects have an "" type and and empty property map.
+     */
+    Formulation():type(std::string()), parameters(geojson::PropertyMap()){}
```

Comment:
```suggestion
    Formulation() = default;
```

Reply:
Simplified in ca837b564057fc79922f55924fa472441bd3c1e1